src/components/App/WorkspaceSwitcher.vue

```diff
@@ -43,7 +45,12 @@ export default {
         "Current Workspace": this.selectedWorkspace,
       });
       this.setActiveWorkspace(this.selectedWorkspace);
-      this.$router.push({ name: "Home", params: { org: this.activeWorkspace } });
+      this.$router.push({
+        name: "Home",
+        params: { org: this.activeWorkspace },
+      });
+
+      this.setUserSwitchedWorkspace();
```

Comment:
```suggestion
      this.setUserSwitchedWorkspace();
      
      this.$router.push({
        name: "Home",
        params: { org: this.activeWorkspace },
      });
```
Should ideally set the store variable before navigating to Home